dll/win32/comdlg32/filedlg.c

```diff
@@ -1915,7 +1915,11 @@ static LRESULT FILEDLG95_InitControls(HWND hwnd)
   /* change Open to Save */
   if (fodInfos->DlgInfos.dwDlgProp & FODPROP_SAVEDLG)
   {
+#ifdef __REACTOS__
```

Comment:
Fix upstream Wine first?

Reply:
What do you mean ?

Reply:
Usual steps are:
1. Fix Wine
2. Sync ReactOS
3. Resolve Jira

Reply:
Yes, it has been reported to Wine : https://bugs.winehq.org/show_bug.cgi?id=48962

Reply:
Are you unaware of the WineSync process?
Don't you understand the difference between a report filed in a bug tracker and a fix committed in a code repository? Scary...

Reply:
Yes i'm unaware of your processes. Happy to know that you'll help me.

Reply:
I won't do more (clues) than I have already done, especially here. (See private chat.)